lib/main_containers.pm

```diff
@@ -139,9 +139,9 @@ sub load_host_tests_podman {
     loadtest 'containers/podman_quadlet' if is_tumbleweed;
     # https://github.com/containers/podman/issues/5732#issuecomment-610222293
     # exclude rootless podman on public cloud because of cgroups2 special settings
-    unless (is_sle('<15-sp2') || is_openstack || is_public_cloud) {
+    unless (is_openstack || is_public_cloud) {
         loadtest 'containers/rootless_podman';
-        loadtest 'containers/podman_remote' if is_sle '>15-sp2';
+        loadtest 'containers/podman_remote';
```

Comment:
I think that podman-remote should not be excluded on OpenStack and PublicCloud - Ricardo, care to test if we can just run it there?

Reply:
> I think that podman-remote should not be excluded on OpenStack and PublicCloud - Ricardo, care to test if we can just run it there?

I could fix this in a next PR as this requires investigation on why it was disabled there.

Reply:
sounds good!